### DISCUSSION TITLE
Debian Package

### DISCUSSION THREAD

**Discussion by Nilsonfsilva** [ProtonMail/gopenpgp#189] [Q&A]
Hello
I'm trying to link gopenpgp to the official Debian repositories.

But I'm stuck with this error. https://salsa.debian.org/nilsonfsilva/gopenpgp/-/jobs/2947586

Could anyone give me some advice?

I'm waiting!

**Comment by twiss** [comment]
The version of [go-crypto](https://github.com/ProtonMail/go-crypto) is too old. Please use the versions specified in [go.mod](https://github.com/ProtonMail/gopenpgp/blob/main/go.mod).


> I'm waiting!

This doesn't help to get a reply faster.

**Comment by Nilsonfsilva** [comment]
Debian to package something, requires that the dependencies already do from their repositories.

the version of crypto that is in debian is this:
https://tracker.debian.org/pkg/golang-github-protonmail-go-crypto

Does it solve?

Another thing:
This dependency is missing:
golang.org/x/mobile v0.0.0-20200801112145-973feb4309
 I'll have to pack first.


Nilson F. Silva







De: Daniel Huigens ***@***.***>
Enviado: segunda-feira, 4 de julho de 2022 16:30
Para: ProtonMail/gopenpgp ***@***.***>
Cc: Nilsonfsilva ***@***.***>; Author ***@***.***>
Assunto: Re: [ProtonMail/gopenpgp] Debian Package (Discussion #189) 
 

The version of go-crypto is too old. Please use the versions specified in go.mod.
I'm waiting!
This doesn't help to get a reply faster.

**Comment by mdosch** [comment]
I think this can be closed as gopenpgp is packaged for debian bookworm which is planned to be released in June: https://tracker.debian.org/pkg/golang-github-protonmail-gopenpgp
